virtool/config/cli.py

```diff
@@ -74,6 +74,18 @@ def entry():
     type=str,
     default="https://9a2f8d1a3f7a431e873207a70ef3d44d:ca6db07b82934005beceae93560a6794@sentry.io/220532",
 )
+@click.option(
+    "--fga-api-scheme",
+    help="The OpenFGA API scheme",
+    type=str,
+    default="http",
```

Comment:
Default should actually be `https`. This is the default in the docs for OpenFGA, I believe.

Your default value should always be secure. Let people opt into less secure methods.